torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -49,7 +49,16 @@ namespace jit {
 
 namespace {
 
-template<class T>
+// XXX: This function is to specialize IValue for tensor type in
```

Comment:
This is a copy/paste that propagates a comment that is incorrect in this context. Let's just inline this instead:

```
auto gradient_ivalue = pop(stack);
auto gradient = gradient_ivalue.isNone() ? at::Tensor() : gradient_ivalue.toTensor()
```